File: Bodloguud_spoj/7Bodlogo.cpp
#include<iostream>
#include<math.h>
using namespace std;

int main()
{
    int n;
    float a, p = 1;
    cin >> n;
    for (int i = 1; i <= n; i++)
    {
        a = 1 + 1 / (i * i);
        p = p * a;
    }
    cout << p;
}

File: chuhal/javascript bodloguud/7Bodlogo.js
let a = 99999, p, l = 0, o = 0, k, m, y = true;
m = a;
while (a > 0) {
    k = a % 10;
    o = o + k;
    a = a / 10;
    p = Math.floor(a);
    a = p;
}
console.log(o);